garage/contrib/ros/docker/sawyer/Dockerfile.gpu

```diff
@@ -0,0 +1,235 @@
+ARG DOCKER_FROM=ubuntu:16.04
+
```

Comment:
This file is a mess of copy/pastes. Please consolidate it.

Please base Dockerfiles meant to be run interactively on [tini](https://hub.docker.com/r/krallin/ubuntu-tini/)

Please move this file into docker/ and make it Dockerfile.contrib-ros-gpu